src/Components/Facility/AssetCreate.tsx

```diff
@@ -875,25 +876,26 @@ const AssetCreate = (props: AssetProps) => {
                       ref={fieldRef["last_serviced_on"]}
                       data-testid="asset-last-serviced-on-input"
                     >
-                      <FieldLabel>Last Serviced On</FieldLabel>
-                      <DateInputV2
+                      <DateFormField
+                        label="Last Serviced On"
```

Comment:
You should add translations and use it here instead of suppressing the eslint warning.